webpack/assets/javascripts/react_app/common/helpers.js

```diff
@@ -6,4 +6,12 @@ export default {
     });
   },
   noop: Function.prototype, // empty function
+  urlBuilder(controller, action, id = undefined) {
+    return `/${controller}/${id ? `${id}/` : ''}${action}`;
+  },
+  getQueryParam(query) {
+    const urlParams = new URLSearchParams(window.location.search);
```

Comment:
how is this different from the urijs lib that we currently use? - also does this work across browsers?

Reply:
It becomes a standard interface, most of the browsers support it, and there's a [polyfill](https://github.com/WebReflection/url-search-params)

Reply:
should we replace than the urijs?